src/utils/pivotal/client.ts

```diff
@@ -164,11 +185,11 @@ export default class PivotalClient {
    * Create a story in the current project.
    * @param {PivotalStory} story
    */
-  async createStory(story: PivotalStory) {
+  async createStory(story: PivotalStory, project: string) {
```

Comment:
lets add new param to jsdoc?
```suggestion
  async createStory(story: PivotalStory, projectId: string) {
```